hdfs/util.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, consumer):
     self._queue = None
     self._reader = None
     self._err = None
+    self.__iter__ = None # __iter__ attribute expected by pandas to write csv file since version 0.24.
```

Comment:
Can we make it a class attribute? It isn't touched by instances.

```python
class AsyncWriter(object):

  # ...

  # Expected by pandas to write csv files (https://github.com/mtth/hdfs/pull/130).
  __iter__ = None
```

Reply:
Yep, done